src/XMLexport.cpp

```diff
@@ -255,8 +255,8 @@ bool XMLexport::writeModuleXML(QIODevice* device, QString moduleName)
             if (!(*it) || (*it)->mPackageName != moduleName) {
                 continue;
             }
-            if ((*it)->mModuleMember) {
-                if (!writeKey(*it)) {
+            if( ! (*it)->isTempKey() && (*it)->mModuleMember ) {
```

Comment:
If you have clang-format available and you are using a recent Qt Creator as your IDE you ought to use that "beautifier" on the functions you have edited - that should make the layout more closely match the layout we are _trying_ to adopt throughout our C++ code now  - and make the number of lines of things that have changed smaller... 🤓